helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: prometheus-agent.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: prometheus-agent
+    rules:
+    ## Page Atlas if prometheus agent fails to send samples to MC prometheus.
+    - alert: PrometheusAgentRemoteWriteNotWorking
+      annotations:
+        description: '{{`Prometheus agent remote write is failing.`}}'
+        summary: Prometheus agent fails to send samples to remote write endpoint.
+        opsrecipe: prometheus-agent-remote-write-failed/
+      expr: absent_over_time(up{instance="prometheus-agent",cluster_type="workload_cluster"}[10m])
+      for: 10m
```

Comment:
10mn feels way too short to me, can we increase this 1h ?

Reply:
1h will keep the alert opened for a while though

Reply:
Also, we might fail to get alerted for the workload cluster if we wait that the agent is down for an hour

Reply:
15m  would be fine also

Reply:
I think 15 is too little. If the cluster breaks down in the 10 minute span, we are blinded

Reply:
hmm, i'm thinking this will page all existing clusters though

Reply:
why?

Reply:
Because it's been absent forever 😅

Reply:
It would be nice if we could make it work only on vintage 19 releases by joining with some kind of cluster version metric

Reply:
Why alert would fail, if we move to 1h ? I am just trying to avoid noise here.

Reply:
Think of thé agent as you think of prometheus. If there is no agent, consider thé cluster unmonitored. Paging only After 1 h is dangerous

Reply:
The agent should only monitor kube-controller-manager, kube-scheduler, and kubelet. So we only loose monitoring to those when the agent is down.
Maybe we want to align this with the Heartbeat delay which is [25mn currently](https://github.com/giantswarm/prometheus-meta-operator/blob/674ab4d08254af8d0070044663d3dc40193fb2c2/service/controller/resource/alerting/heartbeat/resource.go#L94)

Reply:
If we try to integrate a heartbeat for the agent on MC?
That will page one prometheus-agent down! 
then this alert with like 1h or 30m!

Reply:
* Write an inhibition to ignore vintage cluster before v19
* Write an inhibition to ignore "previous" CAPI cluster
* Configure this alert to use the inhibition